rootfs/root/.cas-trusted-signing-pub-key

```diff
@@ -0,0 +1,4 @@
+-----BEGIN PUBLIC KEY-----
+MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAE03LvYuz79GTJx4uKp3w6NrSe5JZI
+iBtgzzYi0YQYtZO/r+xFpgDJEa0gLHkXtl94fpqrFiN89In83lzaszbZtA==
+-----END PUBLIC KEY-----
```

Comment:
@pvizeli I'm realizing we're going to have to do this for every container 😩 Probably ok for now but maybe we should just move it to the base image? Over time we want everything to do this right?

Reply:
It's only needed where the check is running from? And that is only the supervisor.

Reply:
Yeah is better to have it on SU roots for now